### PR TITLE
Change issue pathing for users

### PR DESCRIPTION
**Pull request by curquiza**
First test in this repo. If validated, I will apply it in all other repos

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Support questions & other
+    url: https://discord.meilisearch.com/
+    about: Support is not handle here but on our Discord
```
Comment (brunoocasali) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Support is not handled here but on our Discord
```

**Review by brunoocasali — COMMENTED** [review]
I think we will only be able to see the changes after merging it, right?

**Comment by curquiza** [comment]
> I think we will only be able to see the changes after merging it, right?

yes unfortunately

**Comment by brunoocasali** [comment]
bors merge

**Review by brunoocasali — APPROVED** [review]
Good for me! 🤝

**Pull request merged**